### PR TITLE
Text props: fontColor, fontfamily, fontSize, fontStyle, fontWeight

### PR DESCRIPTION
**Pull request by martunjai**
Added support for Text props: fontColor, fontfamily, fontSize, fontStyle, fontWeight.
Added test apps to verify.

### CODE REVIEW

File: ReactSkia/textlayoutmanager/RSkTextLayoutManager.cpp [hunk 1]
```diff
@@ -47,6 +47,32 @@ SkPaint convertTextColor ( SharedColor textColor ) {
    return paint;
 }
 
+SkFontStyle::Slant convertFontStyle (FontStyle fontstyle) {
```
Comment (pprabhu-ng) [inline ReactSkia/textlayoutmanager/RSkTextLayoutManager.cpp]: argument/s to be updated in camel case format

File: ReactSkia/textlayoutmanager/RSkTextLayoutManager.cpp [hunk 2]
```diff
@@ -47,6 +47,32 @@ SkPaint convertTextColor ( SharedColor textColor ) {
...
+    switch(fontstyle){
+        case FontStyle::Italic : return SkFontStyle::kItalic_Slant;
+        case FontStyle::Oblique :
+        case FontStyle::Normal : 
+        default: 
+            return SkFontStyle::kUpright_Slant;
+    }
+}   
+
+int convertFontWeight (FontWeight fontweight) {
```
Comment (pprabhu-ng) [inline ReactSkia/textlayoutmanager/RSkTextLayoutManager.cpp]: argument/s to be updated in camel case format

File: ReactSkia/textlayoutmanager/RSkTextLayoutManager.cpp [hunk 3]
```diff
@@ -103,6 +129,10 @@ uint32_t RSkTextLayoutManager::buildParagraph (AttributedString attributedString
     ParagraphStyle paraStyle;
     auto fontSize = TextAttributes::defaultTextAttributes().fontSize;
     auto fontSizeMultiplier = TextAttributes::defaultTextAttributes().fontSizeMultiplier;
+    
+    int fontWeight = SkFontStyle::kNormal_Weight;
+    int fontWidth = SkFontStyle::kNormal_Width;
```
Comment (pprabhu-ng) [inline ReactSkia/textlayoutmanager/RSkTextLayoutManager.cpp]: fontWidth value can be directly provided as input , probably this variable is not required.

File: packages/react-native-skia/components/text/FontSize.js [hunk 4]
```diff
@@ -0,0 +1,14 @@
+import * as React from 'react';
+import { AppRegistry, Text } from 'react-native';
+
+const SimpleViewApp = React.Node = () => {
+  return (
+    <>
+      <Text style={{ margin: 10 }}>DEFAULT FONT SIZE</Text>
+      <Text style={{ margin: 10, fontSize: 15 }}>FONT SIZE - 15</Text>
+      <Text style={{ margin: 10, fontSize: 50 }}>FONT SIZE - 30</Text>
```
Comment (pprabhu-ng) [inline packages/react-native-skia/components/text/FontSize.js]: Text string to be updated as 50

File: packages/react-native-skia/components/text/Text_Prop_Combo.js [hunk 5]
```diff
@@ -0,0 +1,16 @@
+import * as React from 'react';
+import { AppRegistry, Text } from 'react-native';
+
+const SimpleViewApp = React.Node = () => {
+  return (
+    <>
+      <Text style={{ color: 'black', fontSize: 30, margin: 10, fontStyle: "normal", fontWeight: "normal", fontFamily: "Courier New" }}>color: 'black', fontSize: 30, margin: 10, fontStyle: "normal", fontWeight: "normal", fontFamily: "Courier New"</Text>
+      <Text style={{ color: 'blue', fontSize: 20, margin: 10, fontStyle: "normal", fontWeight: "bold", fontFamily: "Ariel" }}>color: 'blue', fontSize: 20, margin: 10, fontStyle: "normal", fontWeight: "bold", fontFamily: "Ariel"</Text>
```
Comment (pprabhu-ng) [inline packages/react-native-skia/components/text/Text_Prop_Combo.js]: fontFamily typo for Arial

File: packages/react-native-skia/components/text/FontSize.js (hunk 4 above)
Comment (martunjai) [reply packages/react-native-skia/components/text/FontSize.js]: Updated to 50

File: ReactSkia/textlayoutmanager/RSkTextLayoutManager.cpp (hunk 2 above)
Comment (martunjai) [reply ReactSkia/textlayoutmanager/RSkTextLayoutManager.cpp]: updated to camel case

File: ReactSkia/textlayoutmanager/RSkTextLayoutManager.cpp (hunk 1 above)
Comment (martunjai) [reply ReactSkia/textlayoutmanager/RSkTextLayoutManager.cpp]: updated to camel case

File: ReactSkia/textlayoutmanager/RSkTextLayoutManager.cpp (hunk 3 above)
Comment (martunjai) [reply ReactSkia/textlayoutmanager/RSkTextLayoutManager.cpp]: removed variable and provided value directly

**Review by munezbn — COMMENTED** [review]
Looks fine. Just address review comments from other reviewer

File: ReactSkia/textlayoutmanager/RSkTextLayoutManager.cpp [hunk 6]
```diff
@@ -103,6 +129,9 @@ uint32_t RSkTextLayoutManager::buildParagraph (AttributedString attributedString
     ParagraphStyle paraStyle;
     auto fontSize = TextAttributes::defaultTextAttributes().fontSize;
     auto fontSizeMultiplier = TextAttributes::defaultTextAttributes().fontSizeMultiplier;
+    
+    int fontWeight = SkFontStyle::kNormal_Weight;
```
Comment (munezbn) [inline ReactSkia/textlayoutmanager/RSkTextLayoutManager.cpp:132]: It is SkFontStyle::Weight type right ?

Comment (martunjai) [reply ReactSkia/textlayoutmanager/RSkTextLayoutManager.cpp:132]: yes, 
In vscode, it is suggesting short form.

Comment (munezbn) [reply ReactSkia/textlayoutmanager/RSkTextLayoutManager.cpp:132]: Then better to use SkFontStyle::Weight.  In future if they change the data type in skia then this code will not be compatible.

**Review by pprabhu-ng — APPROVED** [review]

**Review by munezbn — APPROVED** [review]

**Review by pprabhu-ng — APPROVED** [review]

**Review by munezbn — APPROVED** [review]

**Pull request merged**